src/components/Employees/index.tsx

```diff
@@ -0,0 +1,170 @@
+import styles from './Employees.module.css';
+import employees from '@components/Employees/employeesData';
+import EmployeeItem from '@components/EmployeeItem';
+import { IEmployeeItem } from '@components/EmployeeItem/types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearch, faTimesCircle } from '@fortawesome/free-solid-svg-icons';
+import { useSearchParams } from 'react-router-dom';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { employeeExists, findFilters } from '@utils/employees';
+
+function Employees() {
+    const { t } = useTranslation();
+    const [searchParams, setSearchParams] = useSearchParams({});
+    const [selectedFilters, setSelectedFilters] = useState<string[]>([]);
+    const filters = [
+        'Sensei',
+        'Front-End',
+        'Back-End',
+        'QA',
+        'DevOps',
+        'Full-Stack',
+        t('description.available'),
+    ];
```

Comment:
Everything that's inside the function component's body re-binds on each re-render, so when we have constants it's better to defined them outside of the function body, it can be in the same file, but to make it more readable you can create a new file inside `src/constants/employees.ts` and move it there